Pipeline/PackageBuildOutputs/README.md

```diff
@@ -586,6 +612,27 @@ As an example, you can invoke the generation of an IBM Concert Build manifest wi
 
 By default, the IBM Concert Build manifest file is generated in the `tempPackageDir` and named `concert_build_manifest.yaml`. 
 
+## Publishing interfaces
+
+The CLI option `applicationFolderPath` is used to point to the application's Git repository on z/OS Unix System Services. When specified, this parameter enables the PackageBuildOutputs with additional capabilities, like the packaging of object decks and Include Files based on their usage.
+
+When this CLI option is provided, the script will search for an [Application Descriptor file](https://github.com/IBM/dbb-git-migration-modeler/?tab=readme-ov-file#output-files), named `applicationDescriptor.yml` and located at the root level of the application's Git repository. This file contains information on the usage of each artifact of the application: programs can be "internal submodules" or "service submodule" if statically linked by other programs, include files can be public or shared, if referenced by programs from other applications.
+
+If an Application Descriptor file is found, the required information is leveraged, to include in the created archive:
```

Comment:
This section needs an update on the the separation of `lib` and `include`

Reply:
Updated